bin/console

```diff
@@ -14,6 +14,8 @@ set_time_limit(0);
 
 require __DIR__.'/../vendor/autoload.php';
 require_once __DIR__.'/../autoload.php';
+require_once __DIR__.'/../config/bootstrap.php';
```

Comment:
I'm not fan of adding more and more loading in the console, this should be independent

Reply:
if only we could.. but then we can't call the `Configuration::get` function, this is a nightmare!

Reply:
It works! thanks
This way the PR modifies much less things (I left a few bug fixes nevertheless)

Reply:
removed thanks to your suggestion!